src/OpenTelemetry.PersistentStorage.FileSystem/README.md

```diff
@@ -14,40 +14,52 @@ exporters to improve the reliability of data delivery.
 dotnet add package OpenTelemetry.PersistentStorage.FileSystem
 ```
 
-## Basic Usage
+## Usage
 
 ### Setup FileBlobProvider
 
 ```csharp
-using var persistentBlobProvider = new FileBlobProvider("test");
+using var persistentBlobProvider = new FileBlobProvider("@C:\temp");
 ```
 
+#### Configuration
+
 Following is the complete list of configurable options that can be used to set
 up FileBlobProvider:
 
-* `path`: Sets folder location where blobs are stored.
+##### path
+
+Sets folder location where blobs are stored.
+
+##### maxSizeInBytes
+
+Maximum allowed folder size. Default is `50 MB`.
```

Comment:
What happens when the max is reached? New blobs/files are dropped/ignored? Is this logged anywhere?

Reply:
New blobs are ignored. We do log it.